packages/tables/backend/sirius-components-collaborative-tables/src/main/resources/schema/table.graphqls

```diff
@@ -0,0 +1,128 @@
+type Table {
+  id: ID!
+  metadata: RepresentationMetadata!
+  targetObjectId: ID!
+  columns: [Column!]!
+  lines: [Line!]!
+}
+
+type Column {
+  id: ID!
+  label: String!
+}
+
+type Line {
+  id: ID!
+  targetObjectId: ID!
+  cells: [Cell!]!
+}
+
+interface Cell {
+  id: ID!
+  parentLineId: ID!
+  columnId: ID!
+  cellType: String!
+}
+
+type TextfieldCell implements Cell {
```

Comment:
What about the very first use case, a non editable LabelCell?

Reply:
I did not need it for this iteration. I have already done too much not needed in this iteration.
Of course, a non-editable LabelCell is useful but will be done in a next iteration.